src/api/app/models/staging/staging_project.rb

```diff
@@ -8,6 +8,21 @@ def staging_identifier
       name[/.*:Staging:(.*)/, 1]
     end
 
+    def classified_requests
+      requests = (requests_to_review + staged_requests.includes(:reviews)).uniq.map do |request|
+        {
+          number: request.number,
+          state: request.state,
+          package: request.first_target_package,
+          request_type: request.bs_request_actions.first.type,
+          missing_reviews: missing_reviews.select { |review| review[:request] == request.number },
+          tracked: requests_to_review.include?(request)
+        }
+      end
+
+      requests.sort_by { |request| request[:package] }
```

Comment:
Using `sort` should be more efficient here

Reply:
How? The sort depends on the method `first_target_package` from `BsRequest`.

Reply:
You should do:
```
requests.sort { |request1, request2| request1[:package] <=> request2[:package] }

Reply:
That triggered a Rubocop performance cop:

```ruby
app/models/staging/staging_project.rb:23:16: C: Performance/CompareWithBlock: Use sort_by { |a| a[:package] } instead of sort { |request1, request2| request1[:package] <=> request2[:package] }.
      requests.sort { |request1, request2| request1[:package] <=> request2[:package] }
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

So we gone change it back.

Reply:
loving how Ana and rubocop tell different stories :)

Reply:
@coolo, I measured the time, Rubocop is right, I was wrong :cry: The documentation for the `store_by` method is a bit confusing...